scripts/predicting_the_past_import.py

```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 import sqlite3 as sql
 from fuzzywuzzy import fuzz
-from pyne import nucname as nn
+# from pyne import nucname as nn
```

Comment:
This change was made to avoid having to install pyne on my new machine. I found that as long as I didn't use any functions that use `nn` then no errors were raised. `nn` is used for getting the recipes and building the `xml` files for the recipes. The recipes files created through this system are already part of the repo, so I did not have to run this part of the code. I will reset this file back to what it was.